frame/instrumental/src/mock/runtime.rs

```diff
@@ -319,6 +347,7 @@ frame_support::construct_runtime!(
 		Assets: pallet_assets::{Pallet, Call, Storage},
 		GovernanceRegistry: pallet_governance_registry::{Pallet, Call, Storage, Event<T>},
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage},
+		CollectiveInstrumental: pallet_collective::<Instance1>::{Pallet, Call, Event<T>, Origin<T>, Config<T>},
```

Comment:
You created the type (`type InstrumentalPabloCollective = pallet_collective::Instance1;`) above, but here you use `Instance1`. Maybe `InstrumentalPabloCollective` will be more suitable here. What do you think?

Reply:
I can't use in `construct_runtime!` the same name which was used for type annotation.

Reply:
I can rename it something like `CollectivePabloStrategyInstrumental`